lambda/update/attorney_opt_out.go

```diff
@@ -37,9 +40,20 @@ func validateAttorneyOptOut(update shared.Update) (AttorneyOptOut, []shared.Fiel
 		return AttorneyOptOut{}, []shared.FieldError{{Source: "/changes", Detail: "expected empty"}}
 	}
 
-	if update.AuthorUID() == "" {
-		return AttorneyOptOut{}, []shared.FieldError{{Source: "/update", Detail: "author UID missing from URN"}}
+	author := update.Author.Details()
+
+	uidErrors := validate.All(
+		validate.UUID("/update/author/uid", author.UID),
+		validate.UUID("/update/subject", update.Subject),
```

Comment:
this should be `/subject` to match the JSON given, and I think maybe just ignore validation on `author.UID`?

Reply:
Yep makes sense 👍